app/config.c

```diff
@@ -321,9 +360,17 @@ static void test_selection_menu(void)
         display_enabled(POP_R+12, i, test_list[i].enabled);
     }
 
+    bool tty_update = true;
     bool exit_menu = false;
     while (!exit_menu) {
         bool changed = false;
+
+        if (enable_tty && tty_update) {
```

Comment:
This block is repeated multiple times across this file, so maybe there should be a macro for that ?

Reply:
Alternatively, change all occurrences of `tty_update = true` to `tty_update = enable_tty`. then you can simply use `if (tty_update)`.

Reply:
Yes!